### PR TITLE
[DOC-66] copy edit pass on Operations

### PR DESCRIPTION
**Pull request by rsowald**

### CODE REVIEW

File: docs/operations.md [hunk 1]
```diff
@@ -158,17 +158,18 @@ Only chunks whose weight meets the threshold will be executed.
 
 ### Cleaning phase
 
-The final phase removes data from the scavenge database that is no longer needed.
+This final phase removes data from the scavenge database that is no longer needed.
 
 ## Scavenging HTTP options
```
Comment (cdevarenne) [inline docs/operations.md:163]:
```suggestion
## Starting a scavenge
```

File: docs/operations.md [hunk 2]
```diff
@@ -291,47 +299,52 @@ If the accumulation phase is reporting a lot of cache misses, it may benefit fro
...
+In EventStoreDB, events are immutable and cannot be changed after they are written. Usually, when you have an event with data that needs to be deleted you should take the following steps:
 
-- Rewrite the stream to a new stream without the data to be removed
-- Delete the old stream
-- Run a scavenge to remove the data from disk on each node in turn
+- Rewrite the stream to a new stream without the data to be removed.
+- Delete the old stream.
+- Run a scavenge to remove the data from disk on each node in turn.
 
 If this procedure is impractical in your use case, then you can use the redaction tool (the "redactor") to blank out the data of specific events.
```
Comment (cdevarenne) [inline docs/operations.md:315]:
```suggestion
If this procedure is impractical for your use case, then you can use the redaction tool (the "redactor") to blank out the data of specific events.
```

**Review by cdevarenne — CHANGES_REQUESTED** [review]
In genral, because it's a linux command `cron` is not capitalized.

File: docs/operations.md [hunk 3]
```diff
@@ -94,14 +94,15 @@ Do not take [file-copy](#regular-file-copy) backups while scavenge is running. S
...
+
 - How often you delete streams.
 - How you set `$maxAge`, `$maxCount` or `$tb` metadata on your streams.
 - How important freeing the disk space is to you.
 - Your requirements around GDPR.
 
 You can tell from the scavenge output in the logs and streams how much data it is removing. This can help guide how frequently to scavenge.
 
-You can set up a scheduled task, for example using cron or Windows Scheduler, to trigger a scavenge as often as you need.
+You can set up a scheduled task, for example using [Cron](https://cron.com/) or [Windows Task Scheduler](https://learn.microsoft.com/en-us/windows/win32/taskschd/task-scheduler-start-page), to trigger a scavenge as often as you need.
```
[the remark this answers is not in the document]
Comment (rsowald) [reply docs/operations.md:105]: 🤔 isn't Cron a proper noun? Is this not referring to Cron Calendar?

Comment (cdevarenne) [inline docs/operations.md:105]:
```suggestion
You can set up a scheduled task, for example using cron on Linux or [Windows Task Scheduler](https://learn.microsoft.com/en-us/windows/win32/taskschd/task-scheduler-start-page), to trigger a scavenge as often as you need.
```

**Review by cdevarenne — CHANGES_REQUESTED** [review]

**Review by cdevarenne — DISMISSED** [review]

**Review by cdevarenne — APPROVED** [review]

**Review by rmalik-es — APPROVED** [review]

**Pull request merged**